processor/notices/src/main/java/org/mobilitydata/gtfsvalidator/processor/notices/NoticesProcessor.java

```diff
@@ -52,31 +52,35 @@ public SourceVersion getSupportedSourceVersion() {
 
   @Override
   public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment roundEnv) {
-    for (TypeElement element :
-        typesIn(roundEnv.getElementsAnnotatedWith(GtfsValidationNotice.class))) {
-      Optional<NoticeDocComments> comments = docCommentsFactory.create(element);
-      if (comments.isEmpty()) {
-        continue;
-      }
+    try {
+      for (TypeElement element :
+          typesIn(roundEnv.getElementsAnnotatedWith(GtfsValidationNotice.class))) {
+        Optional<NoticeDocComments> comments = docCommentsFactory.create(element);
+        if (comments.isEmpty()) {
+          continue;
+        }
 
-      PackageElement packageElement = processingEnv.getElementUtils().getPackageOf(element);
-      String resourceName = NoticeDocComments.getResourceNameForTypeElement(element);
+        PackageElement packageElement = processingEnv.getElementUtils().getPackageOf(element);
+        String resourceName = NoticeDocComments.getResourceNameForTypeElement(element);
 
-      try {
-        FileObject resource =
-            processingEnv
-                .getFiler()
-                .createResource(
-                    StandardLocation.CLASS_OUTPUT,
-                    packageElement.getQualifiedName(),
-                    resourceName,
-                    element);
-        try (Writer writer = resource.openWriter()) {
-          GSON.toJson(comments.get(), writer);
+        try {
+          FileObject resource =
+              processingEnv
+                  .getFiler()
+                  .createResource(
+                      StandardLocation.CLASS_OUTPUT,
+                      packageElement.getQualifiedName(),
+                      resourceName,
+                      element);
+          try (Writer writer = resource.openWriter()) {
+            GSON.toJson(comments.get(), writer);
+          }
+        } catch (IOException e) {
+          throw new RuntimeException(e);
         }
-      } catch (IOException e) {
-        throw new RuntimeException(e);
       }
+    } catch (Exception e) {
+      System.out.println("Error: " + e);
```

Comment:
👁️ Don't forget to revert these changes before merge if they were added to debug. If there are no other changes in this class other than formatting, I suggestion to revert all changes to reduce noise in this already large PR.

Reply:
This try catch was added by you. Was it only for debugging?

Reply:
Yes, it was due to an exception regarding the null tables, but that is not the case now.

Reply:
Reverted